packages/react-components/src/components/chart/store/store.ts

```diff
@@ -3,16 +3,22 @@ import { devtools, persist } from 'zustand/middleware';
 import { createMultiYAxisSlice, MultiYAxisState } from './multiYAxis';
 import { createDataStreamsSlice, DataStreamsState } from './contextDataStreams';
 
-export type StateData = DataStreamsState & MultiYAxisState;
-export const createChartStore = () =>
+export type StateData = DataStreamsState &
+  MultiYAxisState & { chartId: string; name: string };
+
+export const createChartStore = (
+  initProps: Partial<StateData> & Required<Pick<Partial<StateData>, 'chartId'>>
+) =>
   create<StateData>()(
     devtools(
       persist(
         (...args) => ({
+          ...initProps,
+          name: initProps.chartId,
           ...createMultiYAxisSlice(...args),
           ...createDataStreamsSlice(...args),
         }),
-        { name: 'chartStore' }
+        { name: `${initProps?.chartId}` }
```

Comment:
Can we prefix this with `chart-store-<id>` or something similar? It will help in the future looking through local storage in devtools to know what we're looking at since the ids are just random guids.